RecoTracker/DeDx/plugins/DeDxEstimatorProducer.cc

```diff
@@ -116,48 +112,27 @@ void DeDxEstimatorProducer::produce(edm::Event& iEvent, const edm::EventSetup& i
   edm::Handle<reco::TrackCollection> trackCollectionHandle;
   iEvent.getByToken(m_tracksTag,trackCollectionHandle);
 
-  Handle<TrajTrackAssociationCollection> trajTrackAssociationHandle;
-  if(useTrajectory)iEvent.getByToken(m_trajTrackAssociationTag, trajTrackAssociationHandle);
 
   std::vector<DeDxData> dedxEstimate( trackCollectionHandle->size() );
 
-  TrajTrackAssociationCollection::const_iterator cit;
-  if(useTrajectory)cit = trajTrackAssociationHandle->begin();
   for(unsigned int j=0;j<trackCollectionHandle->size();j++){            
      const reco::TrackRef track = reco::TrackRef( trackCollectionHandle.product(), j );
 
      int NClusterSaturating = 0; 
      DeDxHitCollection dedxHits;
-   
-     if(useTrajectory){  //trajectory allows to take into account the local direction of the particle on the module sensor --> muc much better 'dx' measurement
```

Comment:
uhm, I may have misunderstood the removal of this ``` if (useTrajectory ```.
It looks like now the cosine used in computation is the one that was in the useTrajectory, and the other option is removed.

@VinInn please confirm